src/sentry/grouping/strategies/base.py

```diff
@@ -250,8 +289,8 @@ class StrategyConfiguration:
     id: Optional[str] = None
     base: Optional[Type["StrategyConfiguration"]] = None
     config_class = None
-    strategies: Dict[str, Strategy] = {}
-    delegates: Dict[str, Strategy] = {}
+    strategies: Dict[str, Strategy[Any]] = {}
+    delegates: Dict[str, Strategy[Any]] = {}
```

Comment:
same as above `Strategy[ConcreteInterface]`

Reply:
same here, I can't use a generic type here or I would have to make the entire containing class generic over `I: Interface`